### DISCUSSION TITLE
Convert Dolby Vision DV to SDR? (Tonemap)

### DISCUSSION THREAD

**Discussion by fa1rid** [staxrip/staxrip#1366] [Q&A] [answered]
Hello, I read the Wiki of the 5 methods for tone mapping HDR to SDR , but which method is the best? Does it depend on the video source and hardware used?
The Wiki doesn't mention if HDR is same as Dolby Vision.
I have AMD Radeon 780M GPU.

MediaInfo of my file:

```
Video 
ID : 1 
Format : HEVC 
Format/Info : High Efficiency Video Coding 
Format profile : Main 10@L5.1@High 
HDR format : Dolby Vision, Version 1.0, Profile 7.6, dvhe.07.06, BL+EL+RPU, no metadata compression, Blu-ray compatible / SMPTE ST 2086, Version HDR10, HDR10 compatible 
Codec ID : V_MPEGH/ISO/HEVC 
Duration : 1 h 51 min 
Bit rate : 82.6 Mb/s 
Width : 3 840 pixels 
Height : 2 160 pixels 
Display aspect ratio : 16:9 
Frame rate mode : Constant 
Frame rate : 23.976 (24000/1001) FPS 
Color space : YUV 
Chroma subsampling : 4:2:0 (Type 2) 
Bit depth : 10 bits 
Bits/(Pixel*Frame) : 0.416 
Stream size : 64.2 GiB (93%) 
Default : Yes 
Forced : No 
Color range : Limited 
Color primaries : BT.2020 
Transfer characteristics : PQ 
Matrix coefficients : BT.2020 non-constant 
Mastering display color primaries : Display P3 
Mastering display luminance : min: 0.0050 cd/m2, max: 1000 cd/m2 
Maximum Content Light Level : 1000 cd/m2 
Maximum Frame-Average Light Level : 985 cd/m2 

```

I would like to keep two copies, 4k & FHD (both SDR).
Also, to what codecs conversion is possible? e.g. HEVC, H264, AV1
Your help is highly appreciated. Thanks!

**Comment by Dendraspis** [comment, marked as answer]
> which method is the best? 

Depends on your needs and preferences. I prefer `LibPlacebo`. I recommend trying them out and choose what you prefer. 

> Does it depend on the video source and hardware used?

Some plugins like `LibPlacebo` need a corresponding GPU, for example one with `Vulkan` support, yes.

> The Wiki doesn't mention if HDR is same as Dolby Vision.

Because this is sort of basic knowledge when you deal with HDR, I suggest browsing the internet for explanations like:
https://www.digitaltrends.com/home-theater/what-is-dolby-vision-hdr-for-tvs/
https://www.digitaltrends.com/home-theater/dolby-vision-vs-hdr10-vs-hdr10/

> I have AMD Radeon 780M GPU.

Give it a try, I would also have to use a search engine to check if that one has `Vulkan` support.

> I would like to keep two copies, 4k & FHD (both SDR).

Keep in mind, that you might regret that decision in a few months or years. I don't know why you want to want to keep two copies as that doesn't make much sense to me and just wastes disk space, but I would suggest creating `4K HDR` or `4K HDR + FHD SDR`.

> Also, to what codecs conversion is possible? e.g. HEVC, H264, AV1

Personally I recommend `HEVC`, especially when you want to keep HDR, but `AV1` can also be useful, especially without HDR.

**Comment by fa1rid** [comment]
Thanks for the reply.

Is the Wiki for tonemapping up-to-date?
Do I need to change "RPU Conversion Mode"?

Which Tone mapping Function to use?
![image](https://github.com/user-attachments/assets/9915988f-d227-47cc-a2e9-0fa4e72529a1)

Also in your experience, does it result in colors very similar to the SDR version of the original movie?

Thanks.

**Reply by Dendraspis** [reply]
I'm not sure that this tonemap function can use the Dolby Vision metadata.

Hard to tell, because the SDR versions often differ from the HDR versions, but you get a good result. Nevertheless keeping the video in HDR is the way to go unless you don't have good reasons to tonemap it.

**Comment by fa1rid** [comment]
Also how can I test encode part of the video, as I get this error:

![image](https://github.com/user-attachments/assets/64b4e6b8-4b50-47f6-ac21-f21c49f300bb)
